cfgov/cal/views.py

```diff
@@ -122,8 +181,16 @@ def display(request, pdf=False):
             'range_end':range_end,
         })
 
-        if pdf and PDFreactor:
-            return pdf_response(request, context)
+        if pdf:
+            template_name = 'about-us/the-bureau/leadership-calendar/print/index.html'
+            if PDFreactor:
+                return pdf_response(request, context)
```

Comment:
Hmmmm, I'm not sure on this issue . An alternative approach might be to just manually create the PDF. 

https://blog.idrsolutions.com/2013/01/understanding-the-pdf-file-format-overview/#helloworld